proposals/vector_reproject.json

```diff
@@ -44,7 +44,7 @@
         }
     ],
     "returns": {
-        "description": "A vector data cube with geometries converted into the new projection. The reference system of the vector dimension changes, all other dimensions and properties remain unchanged.",
+        "description": "A vector data cube with geometries converted into the new projection. The reference system of the vector dimensions changes all other dimensions and properties remain unchanged.",
```

Comment:
Why did you remove the comma here? I think it's correct an makes the sentence much easier to read/understand.

Reply:
Not sure either, I'll add it again.

```suggestion
        "description": "A vector data cube with geometries converted into the new projection. The reference system of the vector dimensions changes, all other dimensions and properties remain unchanged.",
```

Reply:
I'm wondering, whether having a dimension parameter as in filter_temporal makes sense?!

Reply:
@m-mohr I guess that you commented in the wrong PR?

Reply:
@clausmichele No, it's the right PR. In theory there could be two geometry dimensions, right?

Reply:
But are you talking about the `filter_temporal` process or `vector_reproject`? In the comment of 2 hours ago you mentioned `filter_temporal`.

Reply:
Yes, filter_temporal also has a dimension parameter and I mentioned that as an example for how it could be done in this process as well.

Reply:
Ok sorry, I misunderstood.
Anyway I agree, probably it could be good to have a dimension parameter with the default set to `geometries`.

Reply:
Yes, but defaults to `geometry` ;-)

Reply:
Added the parameter and added the comma.